firstvoices/backend/tests/test_apis/test_category_api.py

```diff
@@ -104,3 +101,155 @@ def test_detail_404(self):
         wrong_endpoint = self.get_detail_endpoint(self.site.slug, "54321")
         response = self.client.get(wrong_endpoint)
         assert response.status_code == 404
+
+    @pytest.mark.django_db
+    def test_contains_word(self):
+        word_entry = DictionaryEntryFactory(site=self.site)
+        category_word = ParentCategoryFactory(site=self.site)
+        category_word.dictionary_entries.add(word_entry)
```

Comment:
For all of these single-type cases, you need to also add some entries to the categories that are going to be filtered out. I don't think this test would fail if even if the code was broken